023_IP_L289/023_IP_L289_unsafe.py

```
@@ -0,0 +1,305 @@
+        data[index]['description'] = description
+    else:
+        new_id = new_id
+        new_section = {
+            "id": new_id,
+            "name": name,
+            "tag": name,
+            "imageUrl": f"http://116.62.60.158/pic?key={new_id}",
+            "description": description
+        }
+        data.append(new_section)
+        print("成功：新增板块")
```

Ruff (0.14.2)

String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)